example/package.json

```diff
@@ -22,11 +22,18 @@
   },
   "devDependencies": {
     "@babel/core": "^7.0.0",
+    "@types/jest": "^26.0.0",
```

Comment:
if you use Jest 27 shouldn't the types also be in v27?

Reply:
nvm they don't exist as of now: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jest/index.d.ts